org.jacoco.core.test/src-java8/org/jacoco/core/test/validation/targets/BadCycleInterface.java

```diff
@@ -16,7 +16,7 @@
 	public interface Base {
 		static final Object BASE_CONST = new Child() {
 			{
-				Stubs.nop("base clinit"); // $line-1$
+				Stubs.nop("base clinit"); // $line-baseclinit$
```

Comment:
@marchof I used numbers to denote an actual execution order. So I'm wondering why not leave them as is? As an alternatives - `$line-1-baseclinit$` or explicit comments.